### ISSUE TITLE
[Bug]: new incompatability with Express VPN

### ISSUE THREAD

**Issue by johntisdale2** [FreeTubeApp/FreeTube#5643]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

v 0.21.3-win-x64-portable
When my VPN(express VPN) is turned off, no bug  occurs and the freetube app functions normally 
When I connect to Express VPN, Every time I click on a video I get the errors: TypeError: Cannot read properties of null (reading 'reason') followed by "falling back to Invidious API" and then the error: "SyntaxError: Unexpected end of JSON input"
I get all three lines on every video.
This problem is new, and freetube worked perfectly well with Express VPN before a week ago.

### Expected Behavior

Video should load over standard local youtube API

### Issue Labels

feature stopped working

### FreeTube Version

v.0.21.3

### Operating System Version

Windows 11

### Installation Method

Portable

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.21.3

### Additional Information

![Screenshot 2024-09-03 133223](https://github.com/user-attachments/assets/d6fdd764-7334-4668-be05-0eb1e4192bdd)

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by HiImTemmy** [comment]
Also getting the same issue when using Nord VPN. While VPN was active, FreeTube fails to load / play videos and they flash from large to small constantly. 

Disabling the VPN results in everything working fine.

Freetube was working fine with Nord VPN until this evening.

**Comment by absidue** [comment]
Duplicate of #5534

TL;DR the VPN IP address that you are using has been blocked by YouTube, you need to use a different IP address or switch to a different VPN provider. As the block is on YouTube's side and isn't caused by FreeTube (presumably someone is mass downloading videos or running an Invidious instance on that same VPN IP address), there is not much we can do from our side other than to add a better error message with the same suggestion ms as what I said in the previous sentence.

**Comment by johntisdale2** [comment]
> Duplicate of #5534
> 
> TL;DR the VPN IP address that you are using has been blocked by YouTube, you need to use a different IP address or switch to a different VPN provider. As the block is on YouTube's side and isn't caused by FreeTube (presumably someone is mass downloading videos or running an Invidious instance on that same VPN IP address), there is not much we can do from our side other than to add a better error message with the same suggestion ms as what I said in the previous sentence.

Are you sure. My VPN IP address works on invidious. If it was blocked by YouTube, wouldn't it be blocked on invidious?

**Comment by absidue** [comment]
@johntisdale2 No, because when you watch something on Invidious, the Invidious server makes the request to YouTube not your computer.
